FMI-CROSS-CHECK-RULES.md

```diff
@@ -274,17 +244,6 @@ Placeholder   | Possible Values
 ```
 
 
```

Comment:
If we remove 9.2.5 we should include something like

"Upon request, importers must provide simulation results for the FMUs that it claims to successfully simulate to the FMI Steering Committee, which reserves the right to reject such results if they differ too much from the reference solution of the exporter.